sensor_msgs/PointCloud2 kinect_cloud
---
bool success
